src/runtime_src/core/tools/common/reports/ReportAiePartitions.cpp

```diff
@@ -61,6 +61,41 @@ populate_aie_partition(const xrt_core::device* device)
   return pt;
 }
 
+std::string
+calculate_col_utilization(const xrt_core::device* device, boost::property_tree::ptree &_pt)
+{
+  xrt_core::query::total_cols::result_type total_device_cols;
+  try {
+    total_device_cols = xrt_core::device_query_default<xrt_core::query::total_cols>(device, 0);
+  }
+  catch (...) {
+    return "N/A"; //Error getting total device cols
```

Comment:
You don't need to do try/catch with device_query_default, check the function definition
```
auto total_device_cols = xrt_core::device_query_default<xrt_core::query::total_cols>(device, 0);
```
This is sufficient